projects/igniteui-angular/src/lib/action-strip/grid-actions/grid-action-button.component.ts

```diff
@@ -14,11 +14,11 @@ export class IgxGridActionButtonComponent {
      *
      * @example
      * ```html
-     *  <igx-grid-action-button (onActionClick)="startEdit($event)"></igx-grid-action-button>
+     *  <igx-grid-action-button (actionClick)="startEdit($event)"></igx-grid-action-button>
      * ```
      */
     @Output()
-    onActionClick = new EventEmitter<Event>();
+    public actionClick = new EventEmitter<Event>();
```

Comment:
Migration?

Reply:
This component is not exposed to users so I think a simple rename should be enough.